src/main/java/com/github/tommyettinger/textra/TextraTooltip.java

```diff
@@ -51,9 +49,14 @@ public TextraTooltip(@Null String text, TooltipManager manager, Skin skin, Strin
 	public TextraTooltip(@Null String text, final TooltipManager manager, TextTooltipStyle style) {
 		super(null, manager);
 
-		TextraLabel label = newLabel(text, style.label);
+		final TextraLabel label = newLabel(text, style.label);
```

Comment:
You're doing this just to mock Nathan Sweet, aren't you. (He's had me remove `final` on local variables before, since it doesn't matter on at least some JVMs, maybe all.) I don't mind this being here, though!